arquillian/appclient/src/main/java/tck/arquillian/protocol/appclient/AppClientMethodExecutor.java

```diff
@@ -117,6 +120,7 @@ public TestResult invoke(TestMethodExecutor testMethodExecutor) {
                     break;
             }
             result.addDescription(description);
+            log.log(Level.FINE, String.format("[%s] %s : %s", result.getStatus(), description, result));
```

Comment:
I still feel like it makes sense to make the level configurable. Meaning, use a level based on the status.